src/basekernels/piecewisepolynomial.jl

```diff
@@ -1,56 +1,65 @@
 """
```

Comment:
```suggestion
@doc raw"""
```
and then not having to escape all the LaTeX backslashes